### ISSUE TITLE
Finding the list of misclassified samples in train/test and validation set

### ISSUE THREAD

**Issue by jaiswati** [ultralytics/yolov5#6393]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello @glenn-jocher,

Is there any method to find the list of  (labelwise) misclassified samples and export it into csv format? 
also, would you like to recommend any method to do so?

Thank you,
@jaiswati

**Comment by glenn-jocher** [comment]
@jaiswati thanks for the feedback! Our roadmap includes better results introspection tools for val results, but unfortunately for now you can only see metrics broken down by class, not by image.

**Comment by jaiswati** [comment]
This response could be the relevant one since this is the PyTorch based
https://discuss.pytorch.org/t/viewing-misclassified-image-predictions/82410
https://chaturangarajapakshe.medium.com/you-can-get-a-list-of-indexes-for-the-misclassified-instances-and-use-that-with-your-dataset-9ed169e34f9c
